SketchUp/sketchup.rb

```diff
@@ -234,7 +232,7 @@ def self.file_new
   #   directory relative to the SketchUp
   #   installation directory.
   #
-  # @return path - the entire path if successful. If unsuccessful,
+  # @return [String, false] the entire path if successful. If unsuccessful,
```

Comment:
This return `nil` in case of failure. (I think the old docs are full of inaccuracies in this file... :( )